EIPS/eip-5528.md

```diff
@@ -4,7 +4,8 @@ title: Refundable Fungible Token
 description: Allows refunds for EIP-20 tokens by escrow smart contract
 author: StartfundInc (@StartfundInc)
 discussions-to: https://ethereum-magicians.org/t/eip-5528-refundable-token-standard/10494
-status: Review
+status: Last Call
+last-call-deadline: 2022-10-16
```

Comment:
I left some comments below for editorial suggestion, mainly the specification section can be restructure for better readability.